### DISCUSSION TITLE
Is it possible to setup a multi-page web app using gitub pages?

### DISCUSSION THREAD

**Discussion by MrDracs** [community/community#134997] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Like I wish to put a small web app that can track your exercise log onto github pages.

**Comment by ayusharma03** [comment, marked as answer]
its eazy, now to set up a multi-page web application on GitHub Pages. particularly for a web app like an exercise log tracker:

firstly, Create and Structure Your Web App, in an organized manner,
your files (HTML, CSS, JS) into folders.

Create a GitHub Repository: Log in to GitHub, create a new repository, and upload your web app files.

Enable GitHub Pages: Go to the repository’s Settings, find GitHub Pages, select the branch with your files, and save.

Access Your Site: Use the provided GitHub Pages URL to view your site.

Update as Needed: Upload new files or edits to GitHub to update your site.

**Reply by MrDracs** [reply]
would it also work for a database based app?

**Reply by ayusharma03** [reply]
sure it will!! only the actual database wont be backed up unless you host it locally and back up on GitHub, though if it is hosted on any of the database server you dont have to worry about it as it will be on the cloud, always there lingering..
